orchestration/Chart.yaml

```diff
@@ -7,6 +7,6 @@ type: application
 version: 0.0.0
 dependencies:
   - name: argo-templates
-    version: 0.0.4
+    version: 0.0.7
```

Comment:
This is up to 0.0.13 now 😱

Reply:
Wow... I will bump it up. And no, I haven't run it yet, but I will give it a try now. It would just be the `apply-orchestration-workflow` script in monster-deploy, right?

Reply:
Yes. You'll need to update the encode.yaml definition to pass in all the new parameters required by your branch.

Reply:
Looks like it worked :) I'm making another PR now for the parameters in encode.yaml in monster-deploy.